flycheck.el

```diff
@@ -7868,6 +7869,25 @@ See https://github.com/processing/processing/wiki/Command-Line"
   ;; This syntax checker needs a file name
   :predicate (lambda () (buffer-file-name)))
 
+(flycheck-define-checker protobuf-protoc
+  "A protobuf syntax checker using the protoc compiler.
+
+See URL `https://developers.google.com/protocol-buffers/'."
+  :command ("protoc" "--error_format" "gcc"
+            (eval (concat "--java_out=" (flycheck-temp-dir-system)))
+            (eval (concat "--proto_path=" (file-name-directory (buffer-file-name))))
```

Comment:
It adds a directory to paths in which protobuf files will be searched to import. Is necessary, doesn't work otherwise.